composer.json

```diff
@@ -26,14 +26,14 @@
         "nelmio/security-bundle": "~1.4",
         "doctrine/doctrine-migrations-bundle": "~1.0.0",
         "doctrine/migrations": "~1.0.0|1.0.0@alpha",
-        "mopa/bootstrap-bundle": "3.0.0-RC2",
+        "mopa/bootstrap-bundle": "3.0.0",
```

Comment:
I really have no idea. This requires going through the entire application and checking whether everything works in modern browsers. Something that requires special attention and not something to do while upgrading other things. Let's revert to RC2?